docs/conf.py

```diff
@@ -23,7 +21,7 @@
 # -- Project information -----------------------------------------------------
 
 project = "RDK"
-copyright = "2017-2022 Amazon.com, Inc. or its affiliates. All Rights Reserved"
+copyright = "2017-2023 Amazon.com, Inc. or its affiliates. All Rights Reserved"
```

Comment:
Realistically, the copyright could simply remain the initial creation date, but since a range was used... let's update it.